ProjectManager/ProjectManager/Common/Database/RealmDatabaseManager.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  RealmDatabaseManager.swift
+//  ProjectManager
+//
+//  Created by seohyeon park on 2022/09/12.
+//
+
+import RealmSwift
+
+class RealmDatabaseManager: DatabaseManager {
+    let realm = try? Realm()
+
+    func createDatabase(data: Object) {
+        guard let realm = realm,
+              ((try? realm.write({ realm.add(data) })) != nil) else {
+            return
+        }
+    }
+
+    func readDatabase() -> [TaskModel] {
```

Comment:
```suggestion
    func read() -> [TaskModel] {
```
객체 이름자체가 Database 가 포함되어 있으니 중복으로 보입니다. 굳이 없어도 될 것 같아요 :)

Reply:
삭제해주었습니다😁